### PR TITLE
Add ceph storage plugin tests for for openshift/console

### PR DESCRIPTION
**Pull request by bipuladh**
- Adds tests that are run conditionally if Ceph code is touched.

### CODE REVIEW

File: ci-operator/config/openshift/console/openshift-console-master.yaml [hunk 1]
```diff
@@ -90,6 +90,25 @@ tests:
...
+      env:
+      - default: m4.2xlarge
+        name: COMPUTE_NODE_TYPE
+      - default: "1"
+        name: ZONES_COUNT
+      from: pipeline:src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 10Gi
```
Comment (droslean) [inline ci-operator/config/openshift/console/openshift-console-master.yaml:110]: The job seems ok now, but I think that 10G is too much. Can we try to rehearse the job with fewer memory request? Maybe 5Gi?

**Review by spadgett — APPROVED** [review]
/lgtm

**Comment by spadgett** [comment]
@bipuladh It looks like this is causing failures on all PRs. I'm going to revert.

https://prow.ci.openshift.org/?repo=openshift%2Fconsole&job=pull-ci-openshift-console-master-ceph-storage-plugin

**Pull request merged**